app/javascript/entrypoints/admin.tsx

```diff
@@ -363,6 +363,32 @@ ready(() => {
   document.querySelectorAll('[data-admin-component]').forEach((element) => {
     void mountReactComponent(element);
   });
+
+  const notes_reply_btn = document.getElementById('notes-reply-btn');
+  if (notes_reply_btn instanceof HTMLTemplateElement) {
+    document
+      .querySelectorAll('.report-notes__item__actions')
+      .forEach((note_actions) => {
+        const reply_btn = notes_reply_btn.content.cloneNode(true);
+        note_actions.prepend(reply_btn);
+      });
+  }
+
+  Rails.delegate(document, '.notes-reply-btn', 'click', (event) => {
+    event.preventDefault();
+    if (!(event.target instanceof HTMLElement)) {
+      return;
+    }
+    const note = event.target.closest('.report-notes__item');
+    const notes = event.target.closest('.report-notes');
+
+    const new_note_content = document.getElementById('instance_note_content');
+    if (!note || !notes || !(new_note_content instanceof HTMLTextAreaElement))
+      return;
+
+    new_note_content.value = `In reply to [${note.querySelector('.username')?.textContent?.trim()}](#${note.id})\n\n${new_note_content.value}`;
+    new_note_content.focus();
+  });
```

Comment:
This is all for the "reply" button functionality — there isn't actually proper "reply" functionality in here, because we'd need to do some sort of "in_reply_to" column for that, which seems a bit heavy handed for now.